### DISCUSSION TITLE
Install from file inactive

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17311] [How to]
Installed new 6.4 and registered. Despite that I cannot install module from file. Where is trick?

best

**Comment by rskrzypczak** [comment]
if the button to install modules from a file is not visible, the only possibility is that you have set `demo` mode in the configuration files.

**Comment by s2b-git** [comment]
Here is: public static $systemMode = 'prod';
Yet it calling for registration which is done properly

**Comment by CHILLTH** [comment]
Same here. It effectively discouraging from testing. Can developer please, let us know how to overcome it?

**Reply by JohnMaverick** [reply]
I hope it not going to be like in vtiger....?

**Comment by s2b-git** [comment]
Hi.. 2 weeks, and system is not registered by developer. Can community have some answer please?

**Reply by ghost** [reply]
Run the following command, it should bypass the registration requirement for installing modules from file. If my answer violates the license please remove it.

sed -i '19s|.*|                {if $MAXUPLOADSIZE < 1}|' '/var/www/html/layouts/basic/modules/Settings/ModuleManager/ImportUserModuleStep1.tpl'

**Reply by mariuszkrzaczkowski** [reply]
you have a problem with registration, read the documentation, it may not have reached us
https://doc.yetiforce.com/administrator-guides/company/problems-with-system-registration/#if-nothing-changes-during-status-check-make-sure-your-server-has-the-correct-configuration
